README.md

```diff
@@ -0,0 +1,63 @@
+# IoT-Connect Yocto Integration
+
+*The following details yocto layers designed to integrate the IoT-Connect C SDK. The end result is a compiled version of the `basic-sample` app provided in the C SDK repo that is installed to an image target. Once said image is flashed to a target, the basic-sample app should successfully run & establish comms with an appropriately setup https://avnet.iotconnect.io/*
```

Comment:
There are a few dozen IoTConnect C SDKs and just saying "IoT-Connect C SDK" is not specific enough. You should also hyperlink to the actual repo fir IoT-Connect Generic C SDK" in the markdown to the repo.

Reply:
hopefully fixed in https://github.com/avnet-iotconnect/iotc-yocto-c-sdk/commit/aa6cee1b6a1b2e43094fd51895f43fbb628a4fef

Reply:
Partially yes. the you are still calling the IoTConnect Generic C SDK "IoTConnect C SDK". Bu that is fine.